src/main/java/apitests/steps/BaseSteps.java

```diff
@@ -0,0 +1,16 @@
+package apitests.steps;
+
+import apitests.clients.PetClient;
+import apitests.model.ResponseDetails;
+
+public class BaseSteps {
```

Comment:
I have created this class as a potential place of some helper methods. For now it is minimal avoiding of code redundancy code but if the project will grow it can help... We can remove it of course if you want.

Reply:
I've tried to apply your suggestions but I loose again the response context between steps and test classes. In this shape it should stay at it is... I can try this approach without Cucumber and only with RestAssured itself. I hope that you are ok with that...